src/builtin_command.cpp

```diff
@@ -103,7 +103,7 @@ maybe_t<int> builtin_command(parser_t &parser, io_streams_t &streams, const wcha
             }
         } else {  // Either find_path explicitly or just quiet.
             wcstring path;
-            if (path_get_path(command_name, &path, parser.vars())) {
+            if (path_get_path(command_name, &path, parser.vars(), true)) {
```

Comment:
Why does this use the absolute path? Currently

```fish
cd /usr/bin
command -s ./env
```

prints "./env". I don't know if that's of much *use*, but it doesn't appear to be obviously *wrong* either?

Reply:
Yeah, I'd keep the current behavior unless there is a reason for changing this.

Reply:
:facepalm: It was meant be be false everywhere apart from `parse_cmd_string`. I put it there by mistake and didn't check it after. Sorry.